File: 程式語言 V2答案檔/Go/309.go
package main
import "fmt"

func compute(array [6]int) float64 {
	min := 99999.99
	for i := 0; i < 3; i++{
		val := float64(array[i]) / float64(array[i+3])
		if min > val{
			min = val
		}
	}
	return min
}

func main(){
	var array [6]int
	for i := 0; i < 6; i++{
		fmt.Scanln(&array[i])
	}
	fmt.Printf("%.3f", compute(array))
}

File: 程式語言 V2答案檔/PHP/309.php
<?php
    function compute($array){
        $min_value = 4499;
        for($i = 0; $i < 3; $i++){
            $min_value = min($min_value, $array[$i]/$array[3+$i]);
        }
        return $min_value;
    }
    $array = new SplFixedArray(6);
    for($i = 0; $i < 6; $i++){
        $array[$i] = intval(fgets(STDIN));
    }
    $min_value = compute($array);
    printf("%.3f\n", $min_value);
?>